project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.1.1
+sbt.version=1.1.4
```

Comment:
Did you mean to change this as part of this commit?

Reply:
Yes (it's a small bugfix bump, I'd just push it here)